components/cards/CardProject.tsx

```diff
@@ -30,10 +30,15 @@ const CardProject = ({
   imageAlt,
 }: CardProjectProps) => {
   return (
-    <Card>
+    <Card
+      sx={{
+        boxShadow:
+          '0px 2px 4px 0px rgba(52, 61, 62, 0.04), 0px 4px 8px 2px rgba(52, 61, 62, 0.04)',
+      }}
+    >
       <CardContent
         sx={{
-          padding: '2rem',
+          padding: { xs: '2rem 1rem', lg: '2rem' },
```

Comment:
is this the fix for this comment? 

<img width="556" alt="image" src="https://github.com/openseattle/open-seattle-website/assets/1039839/f5b56d41-8e4c-474d-8f46-0f55619c53fd">

Reply:
not necessarily, but I did add another commit that does set a maxWidth on the page content. thanks for pointing it out!